### PR TITLE
feat: add zeebe extractors

### PR DESCRIPTION
**Pull request by marstamm**
This PR adds extractors for Camunda 8 diagrams:

- restructures project in platform + zeebe
- Adds extractors for input/output mappings, DMN results and Multi Instance variables

### CODE REVIEW

File: src/zeebe/util/ElementsUtil.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+import { find, forEach, isArray } from 'min-dash';
```
Comment (pinussilvestrus) [inline src/zeebe/util/ElementsUtil.js:1]: Correct me when I'm wrong, but some utils look like duplicated code. Any chance to have a `src/shared` folder to make this reusable across the platforms?

**Comment by barmac** [comment]
Let's update the readme with the new capabilities and APIs.

File: src/platform/index.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import { filter, find, forEach } from 'min-dash';
```
Comment (pinussilvestrus) [inline src/platform/index.js:1]: I'm asking myself whether `platform` is the best term here. I know we are not super aligned across the libraries, but maybe we should at least use `camunda-platform` (as in the properties panel)?

**Comment by barmac** [comment]
The ElementsUtil could be lifted up as we have now two identical files.

**Review by pinussilvestrus — COMMENTED** [review]
+1 What @barmac mentioned, let's update the documentation.

I'm really happy that this project becomes a refresh now :)

File: src/zeebe/util/ElementsUtil.js (hunk 1 above)
Comment (barmac) [reply src/zeebe/util/ElementsUtil.js:1]: The files are identical :)

File: src/index.js [hunk 3]
```diff
@@ -1,79 +1,2 @@
...
-
-  return combineArrays(scopeVariables, parentsScopeVariables);
-}
-
-// helpers ////////////////////
-
-function combineArrays(a, b) {
-  return a.concat(b);
-}
+export * from './platform';
```
Comment (nikku) [inline src/index.js:1]: Shouldn't we simply export the Zeebe related functionality under a different prefix (i.e. `extract-process-variables/zeebe`)? 

I personally like that more, especially given that cases where we use both are likely rare.

Comment (marstamm) [reply src/index.js:1]: Great Idea, let's to that instead! This way, we don't need the clunky `extractZeebeVariables` and can keep the same name scheme

Comment (barmac) [reply src/index.js:1]: I believe it's the way to go, too.

Comment (marstamm) [reply src/index.js]: Implemented in [088547e](https://github.com/bpmn-io/extract-process-variables/pull/20/commits/088547efa4c54035773c874e131170636793e8ca)

File: src/zeebe/util/ElementsUtil.js (hunk 1 above)
Comment (marstamm) [reply src/zeebe/util/ElementsUtil.js]: removed duplicated code in [44b8467](https://github.com/bpmn-io/extract-process-variables/pull/20/commits/44b84678ce58d15430c174fbdc572c7337073d5b)

**Comment by marstamm** [comment]
I adjusted according to the reviews, it is ready for re-review

File: README.md [hunk 4]
```diff
@@ -64,11 +71,12 @@ const scopeVariables = getVariablesForScope('Process_1', rootElement.businessObj
 
 Note that `origin` and `scope` retrieves the full moddle element. The example outputs are reduced due to better readibility.
 
-Note that [camunda-bpmn-moddle](https://github.com/camunda/camunda-bpmn-moddle) descriptors have to be installed.
+Note that [camunda-bpmn-moddle](https://github.com/camunda/camunda-bpmn-moddle) or [zeebe-bpmn-moddle](https://github.com/camunda-cloud/zeebe-bpmn-moddle) descriptors have to be installed.
 
 ## Properties support
 
-We are currently [extracting process variables](https://github.com/bpmn-io/extract-process-variables/tree/main/src/extractors) from the following diagram properties
+### Camunda Platform
```
Comment (barmac) [inline README.md:78]:
```suggestion
### Camunda Platform 7
```

File: README.md [hunk 5]
```diff
@@ -77,10 +85,20 @@ We are currently [extracting process variables](https://github.com/bpmn-io/extra
 * `camunda:errorMessageVariable`
 * `camunda:escalationCodeVariable`
 
+### Zeebe
```
Comment (barmac) [inline README.md:88]:
```suggestion
### Camunda Platform 8

```

File: README.md (hunk 4 above)
Comment (marstamm) [reply README.md]: 👍

**Review by barmac — APPROVED** [review]

**Pull request merged**